pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.evanlennick</groupId>
     <artifactId>retry4j</artifactId>
-    <version>0.15.0-SNAPSHOT</version>
+    <version>0.16.0-SNAPSHOT</version>
```

Comment:
please don't modify this version, even if this is merged the version will still be `0.15.0-SNAPSHOT`

Reply:
sure will revert.